titan-compiler/syntax_errors.lua

```diff
@@ -26,6 +27,108 @@ local errors = {
 
     { label = "MalformedEscape_x",
         msg = "\\x escape sequences must have exactly two hexadecimal digits." },
+    
+    { label = "NameFunc",
+        msg = "Expected a function name after 'function'." },
+    
+    { label = "OParPList",
+        msg = "Expected '(' for the parameter list." },
+    
+    { label = "CParPList",
+        msg = "Expected ')' to close the parameter list." },
+    
+    { label = "ColonFunc",
+        msg = "Expected ':' after the parameter list." },
+    
+    { label = "TypeFunc",
+        msg = "Expected a type in function declaration." },
+    
+    { label = "EndFunc",
+        msg = "Expected 'end' to close the function body." },
+    
+    { label = "AssignVar",
+        msg = "Expected '=' after variable declaration." },
+    
+    { label = "ExpVarDec",
+        msg = "Expected an expression to initialize variable." },
+    
+    { label = "NameRecord",
+        msg = "Expected a record name after 'record'." },
+    
+    { label = "EndRecord",
+        msg = "Expected 'end' to close the record." },
+    
+    { label = "FieldRecord",
+        msg = "Expected a field in record declaration." },
+    
+    { label = "NameImport",
+        msg = "Expected a name after 'local'." },
+   
+   --this label is not thrown in rule 'import' because rule 'toplevelvar'
+   --matches an invalid input like "local bola import"
```

Comment:
I'm confused about this commend. The ASSIGN in the toplevelvar rule is not optional so it shouldn't be able to parse "local bola import" right?